src/assets/constants/common.ts

```diff
@@ -0,0 +1 @@
+export const API_URL = 'https://ma-backend-api.mocintra.com/api/v1/products';
```

Comment:
assets folder should contain only images, fonts. Move constants folder to src folder.